src/unix/linux_like/linux/mod.rs

```diff
@@ -5007,7 +4990,6 @@ pub const VM_HUGETLB_PAGES: ::c_int = 18;
 pub const VM_SWAPPINESS: ::c_int = 19;
 pub const VM_LOWMEM_RESERVE_RATIO: ::c_int = 20;
 pub const VM_MIN_FREE_KBYTES: ::c_int = 21;
-pub const VM_MAX_MAP_COUNT: ::c_int = 22;
```

Comment:
This is not a placeholder constant: it is an input to a function which returns a count.